addons/main/functions/fnc_manageDebugMode.sqf

```diff
@@ -0,0 +1,23 @@
+params ["_debugMode"];
+
+if (_debugMode) then
+{
+    private _loopScriptHandle = [] spawn 
+        {
+            while { true } do
```

Comment:
Better use CBA_fnc_addPerFrameHandler, but I don't really understand the purpose of this loop

Reply:
With growing complexity it becomes more and more difficult to understand what's going on. Therefore I thought it would be a good idea to add a debug mode instead of using individual debug messages in the code. Then I thought we need triggers to execute these debug functions. A simple method is to check wether or not the debug mode is enabled directly in an individual debug function. Another useful way could be to trigger specific debug functions on interval. We will see in the future if this leads to something useful. 

One thing I already have in mind for a debug function is something I saw on lambs danger. The lambs danger debug mode adds an info box above every AI soldier. Something like that with basic information on power/network values could be useful for us.